
10-05-2019 16:05:55 root:ERROR:hhh
Traceback (most recent call last):
  File "C:\Users\17711\OneDrive\PythonCode\老男孩Python全栈开发\day26_约束_自定义异常_加密、日志模块\loging日志.py", line 32, in <module>
    c = a / b
ZeroDivisionError: division by zero
10-05-2019 16:05:55 root:ERROR:Error 除数不能为0
Traceback (most recent call last):
  File "C:\Users\17711\OneDrive\PythonCode\老男孩Python全栈开发\day26_约束_自定义异常_加密、日志模块\loging日志.py", line 32, in <module>
    c = a / b
ZeroDivisionError: division by zero
10-05-2019 16:05:55 root:ERROR:Log除数不能为0
Traceback (most recent call last):
  File "C:\Users\17711\OneDrive\PythonCode\老男孩Python全栈开发\day26_约束_自定义异常_加密、日志模块\loging日志.py", line 32, in <module>
    c = a / b
ZeroDivisionError: division by zero
04-18-2020 21:18:42 root:ERROR:hhh
Traceback (most recent call last):
  File "h:\BaiduNetdiskDownload\OneDrive\PythonCode\Python_Study_oldboy\day26_约束_自定义异常_MD5加密、日志模块\loging日志.py", line 49, in <module>
    c = a / b
ZeroDivisionError: division by zero
04-18-2020 21:18:42 root:ERROR:This is an customer error message
04-18-2020 21:18:42 root:CRITICAL:This is a customer critical message
